script/packagecloud.rb

```diff
@@ -8,9 +8,13 @@
 
 require "json"
 
+packagecloud_ruby_minimum_version = "1.0.4"
 begin
+  gem "packagecloud-ruby", ">=#{packagecloud_ruby_minimum_version}"
   require "packagecloud"
```

Comment:
Wow, huge oops on my end. Sorry about that, @javabrett.

It's fine as is, my suggestion should have omitted the last "require" statement outside of the begin/rescue block. I'm still not a _huge_ fan of using the `LoadError` for control flow, so if you'd like, you could use [`Gem#latest_spec_for`](http://apidock.com/ruby/Gem/latest_spec_for/class) instead, but either works for me 👍